pg_diffix--1.0.0.sql

```diff
@@ -116,6 +116,47 @@ AS $$
   END;
 $$ LANGUAGE plpgsql;
 
+CREATE PROCEDURE assert_column_is_not_aid(table_name text, column_name text)
```

Comment:
We are starting to build up a lot of internal stuff in this file. Should we add an `internal_` prefix to mark them as non-public API?

Also I have another worry. Don't we need to start writing upgrade files for the extension? I.e. `pg_diffix--1.0.0-1.1.0.sql`

Reply:
> Should we add an internal_ prefix to mark them as non-public API?

I think it is sufficient to leave helper functions undocumented. The name is long enough already.
If someone finds out about it by reading the code, lets assume they know what they are doing.

> Don't we need to start writing upgrade files for the extension?

I had the same thought, but decided against it for now, because:

a) we already updated the SQL module with Fir features, so it is simpler to bump the version number when we release Fir instead of moving the differences to another module;
b) managing multiple API versions on the C side will get complicated;
c) the upgrade command still has to be executed manually for each database, which is not a lot simpler than executing the `drop/create extension` combo (configuration state will be kept in both cases).

We will need to create a separate upgrade module when we'll have to transition the configuration state from the old to the new version, but I don't think we're there yet.

Reply:
Ok, I'm also in favor of keeping it simple.